Resources/Prototypes/_NF/Shipyard/monolith.yml

```diff
@@ -0,0 +1,32 @@
+- type: vessel
+  id: MONOLITH
+  name: KL Monolith
+  description: A large and slow universal ship designed for manual upgrades by the crew.
+  price: 80000 #68k before +15% 
+  category: Large
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/monolith.yml
+
+- type: gameMap
+  id: MONOLITH
```

Comment:
Please use proper naming conventions. You have this all uppercased in random spots and it doesnt match either your own naming conventions and also doesnt match our naming standards. Please do not do this.